### PR TITLE
#77 Re-add updated about page

### LINKED ISSUE #77: Re-add updated about page
Update the about page explaining the current caveats, known issues, and upcoming improvements. @cawarren to provide final copy.

Once this is released, update recidiviz.org/covid with information and instructions about using the web model to augment/replace instructions about the Excel model.

### PR DESCRIPTION
**Pull request by sokunthel**
## Description of the change

> Re-add updated about page
image

## Type of change

- [ ] Bug fix (non-breaking change that fixes an issue)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Configuration change (adjusts configuration to achieve some end related to functionality, development, performance, or security)

## Related issues

Closes #77

### Development

These boxes should be checked by the submitter prior to merging:

- [x] Manual testing has been performed locally

### Code review

These boxes should be checked by reviewers prior to merging:

- [x] This pull request has a descriptive title and information useful to a reviewer
- [x] This pull request has been moved out of a Draft state, has no "Work In Progress" label, and has assigned reviewers
- [x] Potential security implications or infrastructural changes have been considered, if relevant

### CODE REVIEW

File: src/page-about/AboutPage.tsx [hunk 1]
```diff
@@ -12,69 +16,266 @@ const AboutPage: React.FC = () => (
...
+              </li>
+              <li className="my-2">
+                Exposed: Residents and staff who have had contact with an
+                individual positive for COVID-19 but are not yet spreading the
+                virus to others
+              </li>
+              <li className="my-2">
+                Infectious: Residents and staff who are have been exposed to
+                COVID-19 and are now capable of spreading it to other
+                individuals
```
Comment (jessex) [inline src/page-about/AboutPage.tsx:64]: To be consistent, let's end this one with a period.

File: src/page-about/AboutPage.tsx [hunk 2]
```diff
@@ -12,69 +16,266 @@ const AboutPage: React.FC = () => (
...
+              .<br />
+              <br />
+              We also suggest joining the COVID-19 Criminal Justice community on
+              Slack (see{" "}
+              <a href="https://www.recidiviz.org/covid">
+                https://www.recidiviz.org/covid
+              </a>
+              ). This is a community of criminal justice department leads and
+              research staff sharing knowledge on how they're approaching the
+              challenges posed by COVID-19."
```
Comment (jessex) [inline src/page-about/AboutPage.tsx:154]: Oops: unnecessary double quote at the end here.

File: src/page-about/AboutPage.tsx [hunk 3]
```diff
@@ -12,69 +16,266 @@ const AboutPage: React.FC = () => (
...
+                illness, whether or not they are hospitalized.
+              </li>
+            </ul>
+            <h1
+              className="text-2xl sm:text-3xl text-left font-display mt-10"
+              id="methodology"
+            >
+              Methodology
+            </h1>
+            <ul className="list-disc list-inside md:list-outside text-base sm:text-lg font-body leading-normal">
```
Comment (jessex) [inline src/page-about/AboutPage.tsx:84]: @sychang do we have links or copy for these 3 methodology items directly below?

**Comment by sychang** [comment]
Thanks, @sokunthel @jessex! We are workshopping the final copy right now and I will ping when it's ready.

File: src/page-about/AboutPage.tsx [hunk 4]
```diff
@@ -12,69 +16,278 @@ const AboutPage: React.FC = () => (
...
+              facilities. The charts and tables project how far into the future
+              cases will peak in your facilities and how high that peak will be.
+              It also forecasts hospital bed needs, and compares this to the
+              number of hospital beds available in your region.
+            </p>
+            <p className="leading-7 text-base sm:text-lg mt-6 mb-0 font-body">
+              If you have any questions, comments, or concerns about the model
+              itself, check the FAQ section below. If you’re interested in
+              getting custom analyses and reporting based on your jurisdiction’s
+              data,or have additional questions we haven’t answered on the site,
```
Comment (jessex) [inline src/page-about/AboutPage.tsx:37]: Nit: space after the comma: "data, or..."

File: src/page-about/AboutPage.tsx [hunk 5]
```diff
@@ -12,69 +16,278 @@ const AboutPage: React.FC = () => (
...
-              tracking those actions that state and county leaders across the
-              country have taken to reduce populations and transmission rate.
-              Why? Because we believe in transparency, and that knowing what’s
-              happening where is helpful to those empowered to take action. In
-              the coming days, we’ll add more data sources to this site intended
-              to aid decision-makers understand what else they can do and what
-              impact those decisions will have both now and in the long run for
-              public health and criminal justice alike.
+              Questions? Feedback? Need help refining this model or fitting it
+              to your system? Email to{" "}
```
Comment (jessex) [inline src/page-about/AboutPage.tsx:250]: "Email to covid@recidiviz.org..." reads a bit strangely. Let's change it to "Reach out to..."

**Pull request merged**